rulesandguidelines.md

```diff
@@ -1,40 +1,27 @@
 ﻿## /r/ADHD posting rules!
 
-1. **[Read the FAQ](/r/ADHD/wiki)** at least once before posting! If something is missing you can [message the moderators](/message/compose?to=%2Fr%2FADHD) to notify us or request permissions to help expand the wiki.
+1. **[Read the FAQ](/r/ADHD/wiki) and [search /r/adhd](/r/ADHD/search?q=adhd+keywords&restrict_sr=on)** before posting!
 
-2. **[Search /r/adhd](/r/ADHD/search?q=adhd+keywords&restrict_sr=on) for previous posts** on the topic ([search via google](http://www.google.com/search?q=site:reddit.com%2Fr%2Fadhd+adderall)).  
+2. **Use a specific title, keep posts brief, and avoid asking "DAE..."** We've all been there and done that. Spend a minute to write a specific question for your title. Short paragraphs, bold text, and bullet points are good. Aim for 1-2 sentence tl;drs.
 
-3. **Use a specific, meaningful title.**  Incomplete or vague post titles will be removed.^(If it's reported!) *(‘Has anyone felt this?’ ‘First day on Adderall’)*   **Do not title your post with DAE.** People with ADHD always have.  Ask a specific question in the title and ask at least two specific questions in your post. There is *ALWAYS* space.
+3. **Avoid asking if you should get diagnosed or if something is a medication side effect.** If in doubt, asking your doctor or pharmacist should take precedence over what internet laypeople have to say.
 
-4. **If possible, keep posts brief.  If not possible, break it up.**  Use short paragraphs, bold text, numbers, and/or bullet points for longer posts, and include a one or two sentence TL;DR summary.
-
-5. **Meme/comic/picture/video** - With a visual post, use a descriptive title, and preferably also include a comment asking a question for discussion or share your own experience... Pure memes (including anything with a 'hash-tag' such as #justadhdthings) should be posted in /r/adhdmeme and will be removed from here.
-
-6. **No posts asking about drug side effects** - Search google or ask in the weekly question thread. (Reasoning in upcoming posting FAQ.)  
+4. **Memes and media content must be part of a greater discussion.** Take fluff and #justadhdthings to /r/adhdmeme. We have enough distractions here!
     
-7. **AMA’s and surveys require prior mod approval.**
-
-8. **We cannot and will not diagnose.**  We are not doctors and you should not trust internet people with something so important.  If you suspect you might have ADHD, please consult a trained professional.  A psychiatrist or ADHD specialist will be able to help you out.
-
-9. **Not sure if you should seek diagnosis?**  Go seek diagnosis.
+5. **Studies, AMAs, Q&A, and similar from healthcare providers and researchers require prior mod approval.** [Please message the mods.](https://www.reddit.com/message/compose?to=%2Fr%2FADHD)
```

Comment:
Probably want to link to the survey requirements page (https://reddit.com/r/adhd/wiki/surveys ; yet another page that needs work).